app/views/layouts/angular-bootstrap/_endpoints_angular.html.haml

```diff
@@ -77,6 +78,17 @@
                           "checkchange"             => "",
                           "prefix"                  => prefix.to_s,
                           "reset-validation-status" => "#{prefix}_auth_status"}
+    - detect = "detectClicked({target: '.detect_button'})"
+    %span
+      %miq-button.detect_button{"ng-if" => detection ? 'true' : 'false',
+                  :name            => _("Detect"),
+                  :on_click        => detect,
```

Comment:
Aaah.. according to https://docs.angularjs.org/api/ng/type/$compile.directive.Attributes , angular accepts dashes, underscores or colons .. so .. yeah, good to know.

But, we always use the dash version, so.. if you don't mind, to prevent potential confusion..